packages/zero-cache/src/services/mutagen/mutagen.ts

```diff
@@ -44,7 +44,7 @@ export interface Mutagen {
   processMutation(
```

Comment:
There is a narrow race with CRUD mutators.

Client c1 is connected with clientSchema X that is compatible with DB schema.
DB schema is updated such that it is no longer compatible with clientSchema X. 
Before this schema change is replicated c1 pushes a mutation using clientSchema X, which mutagen 
will try to execute against the now incompatible DB schema.

Not sure this is worth addressing given mutagen is expected to go away soon.

Reply:
If this race occurs and the schema change was incompatible with the mutation, likely the mutation will just fail with an error and the client be disconnected.  It will try to reconnect and repush.  When the schema change replicates, the client will be disconnected with an error that will cause it to reload, hopefully getting the updated schema.

Reply:
Yeah this is a form of race we already knew about.